addons/web/static/src/views/graph/graph_model.js

```diff
@@ -374,6 +374,14 @@ export class GraphModel extends Model {
         return new DateClasses(arrays);
     }
 
+    /**
+     * @protected
+     * @returns {string}
+     */
+    _getDefaultFilterLabel(fields) {
+        return this.env._t("Undefined");
+    }
```

Comment:
Can you add a test by creating a custom js class to test this changes and check we have the label expected?

You can create a custom js class inside the test like this, for instance:
```js
QUnit.test("apply custom standard action menu (archive)", async function (assert) {
        // add active field on partner model to have archive option
        serverData.models.partner.fields.active = { string: "Active", type: "char", default: true };

        const formView = registry.category("views").get("form");
        class CustomFormController extends formView.Controller {
            getStaticActionMenuItems() {
                const menuItems = super.getStaticActionMenuItems();
                menuItems.archive.callback = () => {
                    assert.step("customArchive");
                };
                return menuItems;
            }
        }
        registry.category("views").add("custom_form", {
            ...formView,
            Controller: CustomFormController,
        });

        await makeView({
            type: "form",
            resModel: "partner",
            serverData,
            resId: 1,
            actionMenus: {},
            arch: `
                <form js_class="custom_form">
                        <field name="active"/>
                        <field name="foo"/>
                </form>`,
        });

        assert.containsOnce(target, ".o_cp_action_menus");

        await toggleActionMenu(target);
        assert.containsOnce(target, ".o_cp_action_menus span:contains(Archive)");

        await toggleMenuItem(target, "Archive");
        assert.verifySteps(["customArchive"]);
    });
```

Thanks to FGE for his review. 🙂